pkg/kube/daemonset_test.go

```diff
@@ -81,7 +81,14 @@ func expectedDaemonSet(instance *sfv1alpha1.SplunkForwarder) *appsv1.DaemonSet {
 									Name:  "SPLUNK_ACCEPT_LICENSE",
 									Value: "yes",
 								},
-							},
+						                {	Name: "HOSTNAME",
```

Comment:
Similar, `go fmt` honestly might just do it too
```suggestion
						                {
						                  Name: "HOSTNAME",
```